spring-boot-admin-server/src/main/java/de/codecentric/boot/admin/config/NotifierConfiguration.java

```diff
@@ -136,6 +129,23 @@ public PagerdutyNotifier pagerdutyNotifier() {
 		}
 	}
 
+
+	@Configuration
+	@ConditionalOnProperty(prefix = "spring.boot.admin.notify.opsgenie", name = "api-key")
+	@AutoConfigureBefore({ NotifierListenerConfiguration.class,
+			CompositeNotifierConfiguration.class })
+	public static class OpsGenieNotifierConfiguration {
+		@Bean
+		@ConditionalOnMissingBean
+		@ConditionalOnProperty(prefix = "spring.boot.admin.notify.opsgenie", name = "enabled", matchIfMissing = true)
```

Comment:
All other notifier-beans are instantiated even if they are disabled please align to that.